model/Hardware/Properties/plannedBuyer.md

```diff
@@ -0,0 +1,17 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# plannedBuyer
+
+## Summary
+
+A buyer is an individual, organization, or entity that plans to purchase goods, services, or assets.
```

Comment:
Same as `Buyer`, because the definition also include the purchasing of services, this has a potential to be shared by other profiles (SaaS?) as well, apart from Hardware.

We may able to have this outside of the Hardware profile (Core?).